spotbugs/etc/messages.xml

```diff
@@ -9442,6 +9442,61 @@ Using floating-point variables should not be used as loop counters, as they are
      ]]>
     </Details>
   </BugPattern>
+
+  <BugPattern type="US_USELESS_SUPPRESSION_ON_CLASS">
+    <ShortDescription>Useless suppression on a class</ShortDescription>
+    <LongDescription>Suppressing annotation on the class {0} is unnecessary</LongDescription>
+    <Details>
+      <![CDATA[<p>
+      Suppressing annotations <code>&amp;SuppressFBWarnings</code> should be removed from the source code as soon as they are no more needed.
```

Comment:
Perhaps this message could be adjusted to make it clear why the annotation is no longer needed, ie the code no longer contains that bug type.

Reply:
Just something to the effect that the the bug type is no longer raised by that code.

Reply:
I've expanded the message for these unnecessary suppressions, let me know if that could be improved!